### PR TITLE
Build for only one platform on kueue-viz-image target.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
Build for only one platform on kueue-viz-image target.

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
Fix for https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/3881/pull-kueue-build-image-main/1869367074451623936.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -343,20 +343,21 @@ kueue-viz-image-build:
 		--build-arg BUILDER_IMAGE=$(BUILDER_IMAGE) \
 		--build-arg CGO_ENABLED=$(CGO_ENABLED) \
 		$(PUSH) \
-		-f ./cmd/experimental/kueue-viz/backend/Dockerfile ./cmd/experimental/kueue-viz/backend; \
+		-f ./cmd/experimental/kueue-viz/backend/Dockerfile ./cmd/experimental/kueue-viz/backend
```
Comment (mimowo) [inline Makefile:346]: is this part of the fix?

Comment (mbobrovskyi) [reply Makefile:346]: Mmm. No. Let me revert it.

Comment (mbobrovskyi) [reply Makefile]: Reverted.

**Comment by mimowo** [comment]
/lgtm
/approve
/hold
IIUC we need to wait for merging https://github.com/kubernetes/test-infra/pull/33995, feel free to cancel the hold if the build passes after that merge

**Comment by tenzen-y** [comment]
I think that it would be worth making the visibility image buildable for linux/arm64. Especially, the linux/arm64 would be useful for local development.

**Comment by tenzen-y** [comment]
> I think that it would be worth making the visibility image buildable for linux/arm64. Especially, the linux/arm64 would be useful for local development.

Opened https://github.com/kubernetes-sigs/kueue/issues/3916

**Pull request merged**